.github/workflows/pr-validation.yml

```diff
@@ -1,18 +1,101 @@
 name: "Fluid PR Validation"
 on:
   pull_request:
-    types: [ opened, synchronize, reopened ]
+    types:
+      - opened # PR is created
+      - synchronize # commits added to PR
+      - reopened # closed PR re-opened
+      - edited # title or body edited, or base branch changed
     branches:
       - main
+      - next
```

Comment:
Probably remove this next doesn't exist anymore and doesn't apply to patching previous releases.